### PR TITLE
Fix export bug

### PR DESCRIPTION
**Pull request by mauroz77**
- Fix the bug presented when exporting mapping entities
- Removing commented out code
- Enable unit test for export method

### CODE REVIEW

File: rest/src/main/java/org/pdxfinder/controllers/MappingsController.java [hunk 1]
```diff
@@ -157,17 +157,10 @@ public MappingsController(
...
-//    }
-
-
   @GetMapping("export")
   @ResponseBody
-  public Object exportMappingData(HttpServletResponse response,
+  public Object exportMappingData(
+      HttpServletResponse response,
       @RequestParam("mq") Optional<String> mappingQuery,
       @RequestParam(value = "mapped-term", defaultValue = "") Optional<String> mappedTermLabel,
```
Comment (Afollet) [inline rest/src/main/java/org/pdxfinder/controllers/MappingsController.java:165]: Any idea why Optionals were used?

**Review by Afollet — APPROVED** [review]

File: rest/src/main/java/org/pdxfinder/controllers/MappingsController.java (hunk 1 above)
Comment (mauroz77) [reply rest/src/main/java/org/pdxfinder/controllers/MappingsController.java:165]: Not really... Maybe they are not necessary here as anyways a defaultValue is provided. I will check if removing them changes anything functionally.

**Comment by mauroz77** [comment]
@Afollet Thanks for the review. I agree the class can be clean up. What do you think about having a different ticket/PR to do that and close this current one?

**Comment by Afollet** [comment]
That's a good question and it is ultimately up to you.

I do think it is good to keep branches small and short lived. Also, it is good to document your work.

 If you want to go ahead and remove all the unnecessary Optionals in the class and refactor all the other methods then it's probably better to do it on a new branch. If you are just going to clean up that method then just put them in here. 

That's all just my opinion though.

**Comment by mauroz77** [comment]
Additional refactoring to be done with https://app.zenhub.com/workspaces/pdxfinder-board-5f0f036188f27e0014a7bdfc/issues/pdxfinder/pdxfinder/472

**Pull request merged**